src/seedu/addressbook/data/person/Name.java

```diff
@@ -60,4 +60,9 @@ public int hashCode() {
         return fullName.hashCode();
     }
 
+    @Override
+    public int compareTo(Name name){
+        return this.fullName.compareToIgnoreCase(name.fullName);
+    }
```

Comment:
Perhaps it would be good to amend commit message to indicate that the default comparator imposes ascending order and is case-insensitive!

Reply:
Line 64 needs a whitespace before `{`:

> Java reserved words should be followed by a white space.

Refer to [CS2113T Java Coding Standard](https://nuscs2113-ay1819s2.github.io/website/coding-standards/java/intermediate.html) - White space within a statement section.